src/components/Modals/RulesModal.vue

```diff
@@ -116,25 +116,130 @@
                             fields will be destroyed. </li>
-                          <li>You cannot hack a Group card.</li>
+                          <li>You cannot hack a Group card or someone with a Firewall card active.</li>
                         </ol>
                         <b>Relation to computer programming:</b>
-                        Sometimes a mean or mischievous computer programmer. known as a <i>hacker</i> will change a computer program
+                        Sometimes a mean or mischievous computer programmer. Known as a <i>hacker</i> will change a computer program
                         to do something it is not supposed to do. Using functions can make it harder for a hacker to change the computer program,
                         which is why Group cards counter Hack cards in the game.
-                      </a>
+
+                      </div>
+                    </li>
+                    <li><h5>Virus</h5>
+                      <div>
+                        <img src="../../../static/cardImg/Virus.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Virus cards let you infect another player's program, cutting their score in half.</li>
+                          <li>When you play a Virus card, you choose which player will be infected. </li>
+                          <li>You cannot infect someone with a AntiVirus card active.</li>
+                        </ol>
+                        <b>Relation to computer programming:</b>
+                        Sometimes a mean or mischievous computer programmer. Known as a <i>hacker</i> will send you a computer program called a <i>virus</i>
+                        to have your computer do something it shouldn't, making your computer slower.
+                        Overclocking your computer increases the rate at which it runs programs, which is why Overclocking cards counter Virus cards in the game.
+                      </div>
+                    </li>
+                    <li><h5>Power Outage</h5>
+                      <div>
+                        <img src="../../../static/cardImg/PowerOutage.png" alt="Instruction 1 Card" class="cardImage">
+                      </div>
+
+                      <div>
+                        <ol>
+                          <li>Power Outage cards let you stop another player from playing instruction cards.</li>
+                          <li>When you play a Power Outage card, you choose which player will be impacted. </li>
+                          <li>You cannot give a power outage to someone with a generator card active.</li>
+                        </ol>
+                        <b>Relation to computer programming:</b>
+                        Sometimes when the weather is bad you can have a power outage. This makes it so that you can't create programs.
```

Comment:
Perhaps say "Sometimes the power to the computer stops (like when someone trips on a cord,  pushes the power button or a tree branch knocks down a power line). If the computer is off, it can't run any programs."